src/gui/optionsdialog.ui

```diff
@@ -1261,6 +1261,44 @@ Manual: Various torrent properties (e.g. save path) must be assigned manually</s
               </layout>
              </widget>
             </item>
+            <item>
+                <widget class="QGroupBox" name="groupExcludedFileNames">
+                    <property name="title">
+                        <string>Excluded file names</string>
+                    </property>
+                    <layout class="QVBoxLayout" name="verticalLayout_311">
+                        <item>
+                            <widget class="QLabel" name="label_1">
+                                <property name="text">
+                                    <string>Filters:</string>
+                                </property>
+                            </widget>
+                        </item>
+                        <item>
+                            <widget class="QPlainTextEdit" name="textExcludedFileNames">
+                            <property name="toolTip">
+                                    <string>Blacklist filtered file names from being downloaded from torrent(s).
+Files matching any of the filters in this list will have their priority automatically set to "Do not download".
+
+Use newlines to separate multiple entries. Can use wildcards as outlined below.
+*: matches zero or more of any characters.
+?: matches any single character.
+[...]: sets of characters can be represented in square brackets.
+
+Examples
+*.exe: filter '.exe' file extension.
+readme.txt: filter exact file name.
+?.txt: filter 'a.txt', 'b.txt' but not 'aa.txt'.
+readme[0-9].txt: filter 'readme1.txt', 'readme2.txt' but not 'readme10.txt'.</string>
+                                </property>
+                                <property name="lineWrapMode">
+                                    <enum>QPlainTextEdit::NoWrap</enum>
+                                </property>
+                            </widget>
+                        </item>
+                    </layout>
+                </widget>
+            </item>
             <item>
              <widget class="QGroupBox" name="groupMailNotification">
```

Comment:
Was this file changed with a text editor? Lines in this PR have 4 spaces indentation instead of the usual 1.

Reply:
This needs to be changed before merging.

Reply:
> This needs to be changed before merging.

Is anyone going to complete this PR?

Reply:
I fixed it.